packages/trade-widget/src/components/AccountBlockie/AccountBlockie.tsx

```diff
@@ -0,0 +1,95 @@
+import { ZeroEx } from '0x.js';
+import { AbiDecoder, BigNumber } from '@0xproject/utils';
+import { Button, Content, Field, Label } from 'bloomer';
+import * as _ from 'lodash';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as Web3 from 'web3';
+
+import { AssetToken } from '../../types';
+
+import { Blockie } from './Blockie';
+
+const ETH_DECIMAL_PLACES = 18;
+
+/**
+ * AccountBlockie properties.
+ */
+interface AccountBlockieProps {
```

Comment:
Can we come up with a better name? `Blockie` doesn't mean anything to most people. Why not: `Identicon`?